src/api-umbrella/web-app/app/models/log_search/elastic_search.rb

```diff
@@ -266,7 +267,7 @@ def aggregate_by_drilldown!(prefix, size = 0)
       :terms => {
         :field => "request_hierarchy",
         :size => size,
-        :include => "^#{Regexp.escape(prefix)}.*",
+        :include => "#{prefix}.*",
```

Comment:
Yes that's the error that took me the longest time to discover : The regex doesn't seems to be supported. It may be a regression in Elasticsearch